recipe/patches/0005-Do-not-trust-JPEG-decoder-to-determine-image-is-CMYK.patch

```diff
@@ -0,0 +1,26 @@
+From f9343cc87028379fa85597475da4b0ef2e915a15 Mon Sep 17 00:00:00 2001
+From: Andrew Murray <radarhere@users.noreply.github.com>
+Date: Wed, 30 Nov 2022 13:49:07 +1100
+Subject: [PATCH 5/5] Do not trust JPEG decoder to determine image is CMYK
```

Comment:
This patch went in `pillow` version `9.4.0` with PR ( https://github.com/python-pillow/Pillow/pull/6767 ). So presumably it is no longer needed as long as the latest version is building